com.unity.cinemachine/Tests/Editor/Unity.Cinemachine.Editor.Tests.asmdef

```diff
@@ -21,6 +21,12 @@
     "defineConstraints": [
         "UNITY_INCLUDE_TESTS"
     ],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.render-pipelines.high-definition",
+            "expression": "14.0.0",
+            "define": "TEST_CINEMACHINE_HDRP"
```

Comment:
I think the define should be CINEMACHINE_HDRP, for consistency with the rest of CM

Reply:
@AntoineCharton Please address this comment